.github/ISSUE_TEMPLATE/integration_help.md

```diff
@@ -5,10 +5,10 @@ about: Do you need assistance with building the Uppy client in your bundler, or
 labels: Not Accepted
 ---
 
-Transloadit is providing Uppy free of charge. If you want you can self-host all components and never pay us a dime. There's docs and tests and your Bug Reports and Feature Requests are always welcome on GitHub. 
+Transloadit is providing Uppy free of charge. If you want, you can self-host all of its components and never pay us a dime. There are docs and tests, and your bug reports and feature requests are always welcome on GitHub. 
 
-There is also a different category of support that we'd like to call Integration Help: making things work for you, that are already reported working for the larger community.
+There is also a different category of support that we like to call Integration Help. It means that you will be taking advantage of all the knowledge already available within the Uppy community, to help you with all kinds of things that have already been reported as working.
```

Comment:
For the three support categories: Bug Reports, Feature Requests, Integration Help, you keep uppercasing for the latter but ditch for the former 2 @AJvanLoon ?

Reply:
Good point. I didn't interpret the first two as support categories. But then I would lean towards using no capitalization at all.

Reply:
can we use sth else, i think it's helpful to emphasize these are the different entities that we later also use in that table

Reply:
What sort of 'something else' do you have in mind? There only the alternative of using capitalization, right? Which would be fine by me as well, in the end

Reply:
Like italic or `<dfn>`? not sure

Reply:
Back to CAPS!